plugins/im-export-breakpoints/_include-media-export-breakpoints.scss

```diff
@@ -0,0 +1,24 @@
+@function im-breakpoints-to-json($target-name) {
+  $json: '';
+  $target-value: map-get($breakpoints, $target-name);
+
+  @each $name, $value in $breakpoints {
+    $json: if($json == '', '{', $json + ',');
+    $state: $target-value >= $value;
+    $json: $json + '"#{$name}":#{$state}';
+  }
+
+  $json: $json + '}';
+
+  @return $json;
+}
+
+@mixin im-export-breakpoints($element: body) {
```

Comment:
Anywhere really. Do you think it should be called automatically since you're actively including the plugin file?